resources/platform/base/resources/element-templates/README.md

```diff
@@ -0,0 +1,7 @@
+# Element Templates Directory
+
+Place element templates into this directory to use them in the Camunda Modeler.
```

Comment:
Proposal: Document additional, local discovery mechanism, too.

Essentially we have: 

* Put templates in modeler directory
* Put templates in local user configuration
* Put templates local to your diagrams (in a `.camunda/element-templates` folder) :arrow_left: recommended!

Reply:
I also believe that a more appropriate link in this documentation is https://docs.camunda.io/docs/components/modeler/desktop-modeler/element-templates/configuring-templates/. It captures all options.

Reply:
Will follow up.